### PR TITLE
Brand refresh updates/selector

### PR DESCRIPTION
**Pull request by RobTobias123**
Updated images (webp, relative dir) and table data and text to reflect latest brand.

### CODE REVIEW

File: docs/components/ns-selector.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Selectors are used to group an image or illustration with a label into a card. T
 
 ### Selector - Illustration
 
-![Selector - Illustration](https://user-images.githubusercontent.com/78355810/115743858-f4552580-a389-11eb-88a9-55bf38d45716.png)
+![Selector - Illustration](images/ns-selector/content-guidance.webp)
```
Comment (PtitBen) [inline docs/components/ns-selector.md:18]: There is a super thin grey line 
image

File: docs/components/ns-selector.md [hunk 2]
```diff
@@ -53,8 +52,7 @@ Selectors are used to group an image or illustration with a label into a card. T
 | Use if you need to support the label with an image | Mix radio and checkboxes |
 | Use if you need to support the label with an illustration | Use if the image or illustration make the selection harder to understand |
 | Use a heading within the content when providing supporting copy | Add too much content, they should not be miniature web pages |
-| Use inline text links into supporting content if necessary | Use without an image or an illustration |
-| Bolden key messages eg. price within heading to highlight | Use imagery within ns-selector with copy embedded inside |
+| Use inline text links into supporting content if necessary | Use imagery within ns-selector with copy embedded inside |
```
Comment (PtitBen) [inline docs/components/ns-selector.md:55]:
```suggestion
| Use inline text links into supporting content if necessary | Use imagery within `ns-selector` with copy embedded inside |
```

File: docs/components/ns-selector.md [hunk 3]
```diff
@@ -67,7 +65,7 @@ Selectors are used to group an image or illustration with a label into a card. T
 * Make sure the paragraph section is added for more description when illustration is used.
 * Make sure the supporting copy is clear and concise.
 * Make sure the labels explain exactly what the user is selecting. If all content is removed other than the label, then the user would still understand what they are selecting.
-* If using the optional Expander, keep the content concise – you can use lists in the content to help. Be mindful that once a heading is expanded it forces the bottom of that selector down and that of any other instances in the row. 
+* If using the optional Expander, keep the content concise – you can use lists in the content to help. Be mindful that once a heading is expanded it forces the bottom of that selector down and that of any other instances in the row.
```
Comment (PtitBen) [inline docs/components/ns-selector.md:68]:
```suggestion
* If using the optional `ns-expander`, keep the content concise – you can use lists in the content to help. Be mindful that once a heading is expanded it forces the bottom of that selector down and that of any other instances in the row.
```

Comment (PtitBen) [inline docs/components/ns-selector.md:68]:
```suggestion
* If using the optional `ns-expander`, keep the content concise – you can use lists in the content to help. Be mindful that once a heading is expanded it forces the bottom of that selector down and that of any other instances in the row.
```

File: docs/components/ns-selector.md (hunk 1 above)
Comment (RobTobias123) [reply docs/components/ns-selector.md:18]: Removed grey lines and replaced content-guidance.webp image.

**Review by PtitBen — APPROVED** [review]

**Pull request merged**